src/config.jl

```diff
@@ -41,9 +41,15 @@ function persistdir!(config::Config)
     end
 end
 
-function nodelog(config::Config, node, message)
+function nodelog(config::Config, node, message; error=nothing)
+    time = now()
+    if error !== nothing
+        @error "[Node $node | $time]: Encountered error: $message" exception=error
+    else
+        @info "[Node $node | $time]: $message"
```

Comment:
This part in particular is iffy; I'm not sure we really want all of this information in the server log, but I'm inclined to try it out for a bit and see how annoying it is.

Reply:
Oh neat, TIL.